repo.go

```diff
@@ -625,10 +625,12 @@ func (r *Repository) downloadAndSaveItem(client Client, downloadingItem *downloa
 	downloadingItem.path = r.fullPath(it.filePath)
 	downloadingItem.pathMu.Unlock()
 
-	// try a few times in case of network trouble
 	var h hash.Hash
 	var x *exif.Exif
-	for i := 0; i < 3; i++ {
+
+	attempt := 1
+
+	for {
```

Comment:
Ah, I see. I get nervous when the loop counter is moved outside the loop declaration because it's too easy to miss a code path, but having it all on this line almost never goes awry. Would it be possible to fix this by declaring `err` outside the loop, or using `=` instead of `:=` at the os.Create line?